tools/conformance/djxl_conformance.cc

```diff
@@ -61,21 +61,40 @@ bool SaveFile(const char* filename, std::vector<uint8_t> data) {
 struct ImageArray {
```

Comment:
Before, it was clear what this ImageArray represented because there wasn't much option. Now it requires some explanation of what the color_channels and num_extra channels are. Do we need three numbers?

Reply:
I updated the code comments in Jon's MR, documented num_channels, as well as improved the documentation of the frames and ec_frames array layout. PTAL